doc/Feature-bootstrap.md

```diff
@@ -0,0 +1,29 @@
+---                                                                                                                                                                                          
```

Comment:
All the `layout` headers have a lot of trailing spaces.

Reply:
>     Liquid Warning: Liquid syntax error (line 3): Unexpected character { in "{{{&nbsp;<bash_code>&nbsp;}}" in Plugin-rpkg-preprocessor.md

You are strict! That is just a warning. I planned to ignore it indefinitely :)
Fixed.